src/Workspaces/Core/Portable/CodeGeneration/CodeGenerationSymbolFactory.cs

```diff
@@ -428,7 +428,8 @@ public static INamespaceSymbol CreateNamespaceSymbol(string name, IList<ISymbol>
             ImmutableArray<IParameterSymbol>? parameters = default,
             ImmutableArray<SyntaxNode> statements = default,
             INamedTypeSymbol containingType = null,
-            ITypeSymbol returnType = null)
+            ITypeSymbol returnType = null,
+            Optional<ImmutableArray<AttributeData>> returnTypeAttributes = default)
```

Comment:
What's the difference between `Optional<T>` and `Nullable<T>` ?

Reply:
Hmm, I guess I could have done Nullable here too. It's usually the thing used for optional parameters like this since it also takes class types too.

Reply:
Going to leave this as is unless I have to touch this PR again.